### ISSUE TITLE
[Feature] Add possibility to run only one test from several parameterized tests

### ISSUE THREAD

**Issue by viraxslot** [microsoft/playwright#19028]
Hi team. It's a proposal for Playwright Test Runner.

I really like parameterized tests, they allow to decrease amount of code:
https://playwright.dev/docs/test-parameterize#parameterized-tests

What I don't like in them: debugging.
Because I need:
1. Set "only" for the `test` function.
2. Comment a part of test data array (sometimes it can be really huge, so scroll up, scroll down).
3. Run the test, debug, fix the problem.
4. Uncomment test data and remove "only".

**Feature request**: it would be really nice to have something like `test.first` and `test.last` functions to easily debug parameterized tests. 
`test.first` runs the first test only (takes the first element from test data array).
`test.last` runs the last only.

So it could look like:
```typescript
const testData = [1, 2, 3, 4, 5];

// should pass
testData.forEach(data => {
  test.first('check array value 1', () => {
    expect(data).toBe(1);
  });
});

// should fail
testData.forEach(data => {
  test.last('check array value 2', () => {
    expect(data).toBe(1);
  });
});
```

I hope it's possible to implement and you'll like the idea :)

**Comment by viraxslot** [comment]
I'm curious why I commented code all this time because I can do `testData.slice(0,1)`, but I'll keep the request as is.
Maybe you have an another workaround :)

**Comment by dgozman** [comment]
> I'm curious why I commented code all this time because I can do `testData.slice(0,1)`, but I'll keep the request as is. Maybe you have an another workaround :)

I was about to suggest `testData.slice(0, 1)`, but then read your comment 😄 

I don't have a better idea for the `forEach` style. If we ever implement [`test.each` or similar](https://github.com/microsoft/playwright/issues/7036), we should make sure it's easy to debug as you say.

**Comment by viraxslot** [comment]
@dgozman yeah, it was an interesting found 😁 I just used to use this approach, and did it out of habit. Thanks for linking the issue.

**Comment by dgozman** [comment]
Closing this in favor of #7036.
